src/disemvowling.js

```diff
@@ -0,0 +1,3 @@
+export default function disemvowling(sentence){
+  return sentence.split('').filter((character)=>!character.match(/[aeiou ]/)).join('')
+}
```

Comment:
The issue specifically states that the input will be all lower case and with no punctuation.

https://github.com/GuildCrafts/web-development-js/issues/144